README.md

```diff
@@ -109,7 +109,7 @@ After installing OSM, [onboard a microservice application](docs/content/docs/tas
 
 ### OSM Usage Patterns
 
-1. [Ingress](docs/content/docs/tasks_usage/ingress.md) and [Egress](docs/content/docs/tasks_usage/egress.md)
+1. [Traffic](docs/content/docs/tasks_usage/traffic/_index.md)
```

Comment:
super tiny nit: could we rename this to traffic management if you also feel it is more descriptive?